lib/salesforce/column.rb

```diff
@@ -101,6 +101,8 @@ def self.typecast(type, value)
         when :double
           begin
             BigDecimal(value.to_s)
+          rescue ArgumentError
```

Comment:
Minor but I think you can use `bigdecimal/util`'s `to_d`. It's the same implementation.
https://github.com/ruby/ruby/blob/ruby_2_5/ext/bigdecimal/lib/bigdecimal/util.rb#L68

Reply:
Good idea! Thanks